### ISSUE TITLE
Sync with streams api changes

### ISSUE THREAD

**Issue by isonmad** [sysapps/tcp-udp-sockets#90]
Is this spec still in progress or is it abandoned now? It still says e.g. "The enqueue() argument of start()", for one, which hasn't been true since whatwg/streams@4a9b2ec5a1a37c9b113ac31949c3cf88582ef994 in March 2015.

**Comment by SEAPUNK** [comment]
I think it's been discontinued.

**Comment by ClaesNilsson** [comment]
That is correct. The W3C SysApps WG has been discontinued and this API proposal is no longer synced with the Streams API.

Sorry for that.

**Comment by isonmad** [comment]
Why isn't there a large bold red text message to that effect at the top of index.html and README.md?

It looks like all the other sysapps repos are similarly missing that information. Their only change since [the decision appears to have been made](https://lists.w3.org/Archives/Public/public-sysapps/2015Apr/0026.html) in 2015 was acknowledging getting money from the European Commission.

**Comment by AMorgaut** [comment]
@isonmad For your information, while the specification is in hold, I started recently to write a polyfill for the UDP part of this API:

> https://github.com/AMorgaut/html5-udp-socket

I found no really convincing cordova plugin and would like to use the same code in cordova as in chrome apps/extensions, firefox, windows, or nodejs apps (even maybe in more classic web browsers via Flash/Java/ActiveX bridges)

I fall in _the enqueue() argument of start()_ "issue", but it was easy to solve as the only change is to use _the enqueue() method of the controller argument of start()_ instead
https://github.com/AMorgaut/html5-udp-socket/blob/master/index.js#L873-L900

I plan to use this opportunity to write tests using the official W3C tools: http://testthewebforward.org/docs/
Which might be a way to contribute to official UDP API tests to https://github.com/w3c/web-platform-tests and maybe revive the work on this API.

PS: if anyone is interesed to contribute, please feel free to join
